news/2016-06-28-osumania-4k-world-cup-2016-registrations-now.md

```diff
@@ -5,11 +5,11 @@ permalink: http://osunews.tumblr.com/post/146609981558/osumania-4k-world-cup-201
 date: 2016-06-28 16:48:48 +0000
 tumblr_url: http://osunews.tumblr.com/post/146609981558/osumania-4k-world-cup-2016-registrations-now
 ---
-We'd like to welcome all of you to this year's official osu!mania 4K World Cup. As usual, the rules have been adjusted from the previous MWC 4K, so please **[read them carefully](https://osu.ppy.sh/help/wiki/Tournaments/MWC/2016/4K#Ruleset)** before registering for this tournament.
+We'd like to welcome all of you to this year's official osu!mania 4K World Cup. As usual, the rules have been adjusted from the previous MWC 4K, so please **[read them carefully](https://osu.ppy.sh/help/wiki/Tournaments/MWC/2016_4K#Ruleset)** before registering for this tournament.
```

Comment:
Nitpick, jump to doesn't work with capital letters.
`#Ruleset` -> `#ruleset`

Reply:
why am i fixing problems with things im not changing Zz